mergify_cli/ci/junit_processing/quarantine.py

```diff
@@ -98,12 +108,30 @@ async def check_and_update_failing_spans(
     if quarantined_tests_spans:
         click.echo("  - 🔒 Quarantined:")
         for qt_span in quarantined_tests_spans:
-            click.echo(f"      · {qt_span.name}")
+            if qt_span.attributes is not None:
+                generate_test_report(
```

Comment:
You should pass directly `qt_span.attributes` and handle the `if None` and `cast` there to avoid repeating the code.